src/ldw-barsPlus.js

```diff
@@ -697,15 +697,15 @@ export default {
         .append("text")
         .attr("class", "ldwtxt")
         .style("opacity", "0")
-        .each(function (d, i) {
+        .each(function (dataObject) {
```

Comment:
thats really generic, its doing foreach on g.texts, is that textNodes?

Reply:
Nope , it is a data object :/
a smaple:
`{
dim1: "Philippines"
dim2: "Philippines"
offset: 0
qElemNumber: 62
qNum: 6139236.168160478
qText: "6139236.2"
qTextPct: ""
}`